src/main/java/com/xpinjection/springboot/dao/valueobject/Recommendation.java

```diff
@@ -0,0 +1,12 @@
+package com.xpinjection.springboot.dao.valueobject;
+
+import lombok.Value;
+
+@Value
+public class Recommendation {
+    private final String instantiator;
```

Comment:
It is better to name it according to business context, in this case it may be "sentence" or "book".

Reply:
That name was just the result of my reversal of your intent. 
It's up to you to refine it